src/Components/ServiceScene/ServiceScene.tsx

```diff
@@ -397,8 +430,29 @@ export class ServiceScene extends SceneObjectBase<ServiceSceneState> {
   }
 
   private subscribeToLogsQuery() {
-    return this.state.$data?.subscribeToState((newState) => {
+    return this.state.$data?.subscribeToState((newState, prevState) => {
       this.updateLoadingState(newState, TabNames.logs);
+      if (newState.data?.state === LoadingState.Done || newState.data?.state === LoadingState.Streaming) {
+        const resultCount = newState.data.series[0].length;
```

Comment:
```suggestion
        const resultCount = newState.data.series[0]?.length ?? 0;
```
Just as a safeguard if there's no series?